src/overdrive.js

```diff
@@ -4,6 +4,19 @@ import ReactDOM  from 'react-dom'
 const renderSubtreeIntoContainer = ReactDOM.unstable_renderSubtreeIntoContainer;
 const components = {};
 
+const prefix = styles => {
+    const propertiesToPrefix = ['transform', 'transformOrigin', 'transition'];
+    const prefix = 'Webkit';
+    const prefixedStyles = {};
+    Object.keys(styles).forEach(property => {
+        if (propertiesToPrefix.indexOf(property) >= 0) {
```

Comment:
Oh, I see. I wasn't paying attention that the `indexOf` is over the array and not per string. Can probably be optimized with and object:
```js
const propertiesToPrefix = {
  transform: true,
  transformOrigin: true,
  transition: true
};

//later
if (propertiesToPrefix[property]) {
   ...
}
```

and the `propertiesToPrefix` can be outside of the actual function so only constructed once.

I'm good with both solutions. WDYT?